### ISSUE TITLE
Question about Process.CreateTime

### ISSUE THREAD

**Issue by mk2** [shirou/gopsutil#50]
Thank you for the great library.

I have a question about Process.CreateTime method.

Please tell me what is calculated in the following code in the method.

I think that the calculation makes time in milliseconds of (seconds since booted + seconds until process started). Not created time in seconds.

My thought is wrong?

- https://github.com/shirou/gopsutil/blob/master/process/process_linux.go#L564
```go
	ctime := ((t / uint64(ClockTicks)) + uint64(bootTime)) * 1000
```

**Comment by shirou** [comment]
Thank you for the comment. You are right. I have fixed that issue at 0921e1f1536769f17ff188a253294adcbc4e2c03.
Thank you!

**Comment by mk2** [comment]
Thanks!
